cdap-ui/app/cdap/components/AbstractWidget/FormInputs/Select/index.tsx

```diff
@@ -160,5 +161,13 @@ const CustomSelect: React.FC<ISelectProps> = ({
 };
 
 (CustomSelect as any).propTypes = WIDGET_PROPTYPES;
+(CustomSelect as any).getWidgetAttributes = () => {
+  return {
+    options: { type: 'ISelectOptions[]|string[]|number[]', required: true },
+    default: { type: 'string', required: false },
+    dense: { type: 'boolean', required: false },
+    inline: { type: 'boolean', required: false },
```

Comment:
Should "native" be here as well (see line 107)? I think it also needs to get added to ISelectWidgetProps

Reply:
Wow, good point! Will add `native` in both `getWidgetAttributes` and `ISelectWidgetProps`.